src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.querySelector('#largeImg');
+const listImg = document.querySelector('#thumbs');
+
+listImg.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const target = e.target;
+
+  if (!target) {
+    return;
+  }
```

Comment:
The check for `target` being null is unnecessary because `e.target` will always be an element. Consider removing this check.